include/aspect/simulator_access.h

```diff
@@ -34,6 +34,7 @@
 #include <deal.II/dofs/dof_handler.h>
 #include <deal.II/fe/fe.h>
 #include <deal.II/fe/mapping_q.h>
+#include <deal.II/multigrid/mg_transfer_matrix_free.h>
```

Comment:
is this needed?

Reply:
No, just left over from an older version. I'll remove it.